lib/supabase/migrations/20251123143855_in_process_sales_table.sql

```diff
@@ -0,0 +1,61 @@
+alter table "public"."in_process_moments" drop constraint "in_process_moments_collection_id_fkey";
+
+alter table "public"."in_process_moments" drop constraint "in_process_moments_collection_id_token_id_unique";
+
+drop index if exists "public"."in_process_moments_collection_id_token_id_unique";
+
+alter table "public"."in_process_moments" drop column "collection_id";
+
+alter table "public"."in_process_moments" add column "collection" uuid not null;
+
+alter table "public"."in_process_moments" add constraint "in_process_moments_collection_fkey" FOREIGN KEY (collection) REFERENCES public.in_process_collections(id) ON UPDATE CASCADE ON DELETE CASCADE not valid;
+
+alter table "public"."in_process_moments" validate constraint "in_process_moments_collection_fkey";
+
+CREATE UNIQUE INDEX in_process_moments_collection_token_id_unique ON public.in_process_moments USING btree (collection, token_id);
+
+alter table "public"."in_process_moments" add constraint "in_process_moments_collection_token_id_unique" UNIQUE using index "in_process_moments_collection_token_id_unique";
+
+create table "public"."in_process_sales" (
+    "id" uuid not null default gen_random_uuid(),
+    "token" uuid not null,
```

Comment:
What do you think about naming this `moment` instead of `token`?

```suggestion
    "moment" uuid not null,
```